### ISSUE TITLE
optimizer.step()、scheduler.step()

### ISSUE THREAD

**Issue by 549506247xxl** [549506247xxl/ArbSR-cuda11.x#2]
警告，optimizer.step()和scheduler.step()的调用顺序需要调整。
